### PR TITLE
add brew instructions

### PR DESCRIPTION
**Pull request by coffeepac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # homebrew-kraken
 Brew tap for kraken
+
+You can install the official build on OS X via Brew by:
+
+```
+brew tap 'samsung-cnct/homebrew-k2cli'
```
Comment (yenicapotediaz) [inline README.md]: if this references the repo, shouldn't it be `samsung-cnct/homebrew-kraken`?

Comment (coffeepac) [reply README.md]: I am unsure.  I don't think we modified the tap registry with homebrew.  I actually just copied this from the kraken repo

Comment (philoserf) [reply README.md]: BTW/FWIW: the single quote is not needed and not common in other projects brew tap instructions.

Comment (yenicapotediaz) [reply README.md]: I tried installing it via `samsung-cnct/homebrew-kraken` and it worked. I think we should update it to that for consistency.

**Pull request merged**